src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala

```diff
@@ -72,9 +72,9 @@ class FileMetadataService(
 
   def deleteFileMetadata(input: DeleteFileMetadataInput, userId: UUID): Future[DeleteFileMetadata] = {
     val propertiesToDelete = descriptionDeletionHandler(input.propertyNames)
+    val consignmentId: UUID = input.consignmentId.getOrElse(throw InputDataException("No consignment id"))
```

Comment:
Can we make `consignmentId` mandatory instead of checking here if it is `None` and then throwing an exception?

https://github.com/nationalarchives/tdr-consignment-api/blob/master/src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/FileMetadataFields.scala#L40

Reply:
Not without causing a breaking change, which we need to avoid.

Reply:
OK